.github/workflows/e2e-tools.yml

```diff
@@ -0,0 +1,90 @@
+name: CI E2E - for tools
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches:
+      - main
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  e2e-tools:
+    runs-on: ubuntu-20.04
+    env:
+      LOG_DIR: /tmp/test_e2e_logs
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # Fetch all history for all tags and branches
+
+    - name: Set up golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.18.1
```

Comment:
don't we need >= 1.18.2 go versions for >=4.11?

Reply:
good point, let me update with a separate commit